config.go

```diff
@@ -34,7 +34,7 @@ var (
 	ClientVersion = "2020-08-20"
 
 	// Agent version to control agent rollover.
-	AgentVersion = "2020-08-17"
```

Comment:
Is it worth rolling over the agent for this? If we don't update this the consequence is just that up hosts won't update the agent, but new hosts will have this updated agent, right? Unless we expect this will actually confuse people.

Reply:
The point of the revision string isn't to selectively update the agent. If we don't update it things will also be confusing if we try to debug since it won't be running the code that you might think it is